src/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+from sqlalchemy import Boolean, Column, ForeignKey, String
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import relationship
+
+from src.service import Base
+
+
+class Projects(Base):
+    __tablename__ = "projects"
+
+    project_id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    name = Column(String, index=True)
+    description = Column(String, index=True)
+    user_projects = relationship("UserProject", back_populates="project")
+    document = relationship("Documents", back_populates="project")
+
+
+class Users(Base):
+    __tablename__ = "users"
+
+    user_id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    name = Column(String, index=True)
+    email = Column(String, unique=True, index=True, nullable=False)
+    password = Column(String, nullable=False)
+    user_projects = relationship("UserProject", back_populates="user", cascade="all, delete-orphan")
```

Comment:
project has many users, right? let's name it appropriately then (back populates)